pkgs/development/python-modules/python-redis-lock/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, redis
+, pytest
+, process-tests
+, pkgs
```

```suggestion
, pkgs
, withDjango ? false, django_redis
```

There's an `extras_require` for this so we can add an argument to enable/disable the feature.